src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/indentation/indentation/Example1.java

```diff
@@ -0,0 +1,57 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="Indentation"/>
+  </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.indentation.indentation;
+
+// xdoc section -- start
+class Example1 {
+    String field = "example";// basicOffset
```

Comment:
Please add space before each `//` and space after to make easy to read.

Reply:
done